frontend/src/main/scala/bloop/Bloop.scala

```diff
@@ -7,9 +7,13 @@ import bloop.io.AbsolutePath
 import bloop.io.Timer.timed
 import bloop.logging.BloopLogger
 
+import jline.console.ConsoleReader
+
 import scala.annotation.tailrec
 
 object Bloop {
+  private final val reader = consoleReader()
```

Comment:
Is it necessary that this is a val? Couldn't it be a local value in `run`? What's the value of having it at the object level?

Reply:
Run is recursively called, and I don't want to create a new instance after each command. Do you prefer that I refactor `run`?